modules/installation-creating-aws-vpc-localzone.adoc

```diff
@@ -0,0 +1,111 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_aws/installing-aws-localzone.adoc
+
+:_content-type: PROCEDURE
+[id="installation-creating-aws-vpc-localzone_{context}"]
+= Creating a VPC that uses local zones in AWS
+
+You must create a Virtual Private Cloud (VPC) in Amazon Web Services (AWS) that uses local zones for your {product-title}
+cluster to use. You can further customize the VPC to meet your requirements, including
+VPN and route tables.
+
+You can use the provided CloudFormation template and a custom parameter file to create a stack of AWS resources that represent the VPC.
```

Comment:
This language was copied over from the other modules that discussed using CF templates, so I think that the current approach is completely acceptable for now! If you want to simplify it into a single command later, we can work on that. (Especially since these are the only CF templates that this method uses!)

Reply:
Let's keep as it for now, mainly as we are already using this.